packages/eslint-plugin/docs/rules/react-require-autocomplete.md

```diff
@@ -0,0 +1,31 @@
+# Require inputs have have autocomplete attributes when necessary
+
+The HTML `autoComplete` helps users to complete filling in forms by using data stored in the browser. This is particularly useful for people with motor disabilities or cognitive impairment who may have difficulties filling out forms online.
```

Comment:
```suggestion
The HTML `autocomplete` attribute helps users to complete filling in forms by using data stored in the browser. This is particularly useful for people with motor disabilities or cognitive impairment who may have difficulties filling out forms online.
```

Reply:
I'm using `autoComplete` with an uppercase `c` here because the rule is targeted towards React, and that the supported attribute has the uppercase letter: https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes.

Do you think I should change the value here in the docs?